include/stdlib.h

```diff
@@ -32,7 +32,9 @@ extern "C" {
 #define EXIT_SUCCESS 0
 #define EXIT_FAILURE 1
 
-#define ATEXIT_MAX 32
+#define ATEXIT_MAX      32
+#define ATEXIT_TYPE     0
```

Comment:
Remainder: please move `ATEXIT_TYPE` and `CXA_ATEXIT_TYPE` to `atexit.c`

Reply:
I deleted `ATEXIT_TYPE` and `CXA_ATEXIT_TYPE`. Instead, I used bool values to indicate what function is registered. `0` if the function doesn't need arguments, '1' otherwise.